autoload/go/util.vim

```diff
@@ -36,3 +36,14 @@ function! go#util#IsWin()
 
     return 0
 endfunction
+
+" StripPath strips the path's last character if it's a path separator.
+" example: '/foo/bar/'  -> '/foo/bar'
+function! go#util#StripPathSep(path)
+	let last_char = strlen(a:path) - 1
+	if a:path[last_char] == go#util#PathSep()
+		return strpart(a:path, 0, last_char)
+	endif
+
+	return a:path
```

Comment:
Yeah the problem is, some files have the indent definition on the bottom, such as:

```
" vim:ts=4:sw=4:et
```

With that, it automatically tries to adapt. However this missing in both `oracle.vim` and `util.vim` (probably it's missing in some other files too). I'll add them whenever I see it's missing :)